src/br/com/jonilson/edigi/model/Book.java

```diff
@@ -104,6 +104,10 @@ private void setPrice(double price) {
         this.price = price;
     }
 
+    public double getPrice() {
+        return price;
```

Comment:
consistência, em alguns lugares você usa o this e em outros não, mesmo se referindo a atributos do objeto. Escolha uma forma e seja consistente.

Reply:
O editor me pegou nessa, sempre prefiro usar o this, fica mais legível o código para outras pessoas.